### PR TITLE
BAB - Add explanations of assistants

### PR DESCRIPTION
**Pull request by CAnorbo**
Fix for #12446

### CODE REVIEW

File: wwwroot/buy-anything.html [hunk 1]
```diff
@@ -89,7 +89,41 @@ <h2 class="title-small green-text d-block">Discover the Future of Shopping</h2>
...
+                                <img src="img/car.png" class="icon-size" alt="All-Purpose Concierge Assistant">
+                                <div class="title my-3 green-text">All-Purpose Concierge Assistant</div>
+                                <p>Choose this option for physical goods and services. Describe exactly what you are looking to buy and get it delivered to you, whether it’s electronics or even a car!</p>
+                            </div>
+                        </div>
+                        <div class="col-lg-4 mb-4">
+                            <div class="feature-box">
+                                <img src="img/plane.png" class="icon-size" alt="Fast Travel Assistant">
+                                <div class="title my-3 green-text">Fast Travel Assistant</div>
+                                <p>If you have a specific flight or hotel picked out and want to pay with Bitcoin, choose the Fast Travel Assistant to confirm your plans quickly.</p>
```
Comment (MarnixCroes) [inline wwwroot/buy-anything.html:113]:
```suggestion
                                <p>If you have a specific flight or hotel picked out, choose the Fast Travel Assistant to confirm your plans quickly.</p>
```
no need to mention

Comment (CAnorbo) [reply wwwroot/buy-anything.html]: Fixed.

File: wwwroot/buy-anything.html [hunk 2]
```diff
@@ -299,7 +333,7 @@ <h2 class="pt-header title-lined left">Discover the Future of Shopping Now</h2>
                                 <a class="btn btn-primary mt-5 mb-5" href="index.html#download">DOWNLOAD WASABI WALLET</a>
                             </div>
                         </div>
-                        
+
```
Comment (yahiheb) [inline wwwroot/buy-anything.html:336]:
```suggestion
```

**Review by yahiheb — CHANGES_REQUESTED** [review]
LGTM, just a small nit.

File: wwwroot/buy-anything.html [hunk 3]
```diff
@@ -79,7 +79,7 @@
                         <div class="col-md-6 order-2 order-md-1">
                             <h1 class="title">Buy Anything with Bitcoin</h1>
                             <h2 class="title-small green-text d-block">Discover the Future of Shopping</h2>
-                            <p>For the 2023 Christmas season, Wasabi Wallet is introducing an exciting feature in partnership with ShopinBit, enabling you to buy anything directly from your wallet using Bitcoin privately. No need for exchanges or revealing personal information.</p>
+                            <p>Wasabi Wallet is introducing an exciting feature in partnership with ShopinBit, enabling you to buy anything directly from your wallet using Bitcoin privately. No need for exchanges or revealing personal information.</p>
```
Comment (yahiheb) [inline wwwroot/buy-anything.html:82]:
```suggestion
                            <p>Wasabi Wallet has introduced an exciting feature in partnership with ShopinBit, enabling you to buy anything directly from your wallet using Bitcoin privately. No need for exchanges or revealing personal information.</p>
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Review by yahiheb — APPROVED** [review]
LGTM

**Review by adamPetho — APPROVED** [review]
LGTM, any reason why this isn't merged in yet?

**Comment by CAnorbo** [comment]
> LGTM, any reason why this isn't merged in yet?

There were some change requests, but it's ready to merge.

**Pull request merged**